content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -179,7 +179,7 @@ gcloud.compute.zone | The zone to use for zonal resources; must be in gcloud.com
   * For the default configuration, you need to choose a location that supports NVIDIA Tesla K80 Accelerators (`nvidia-tesla-k80`). 
     To see which accelerators are available in each zone, run the following command:
```

Comment:
Please update line 180 to clarify what the updated command does.

To see which zones have `nvidia-tesla-k80` accelerators, run the following command: